### PR TITLE
Reword README to add links to conda-forge migration status

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,17 @@
 # miniforge
 [![Build Status](https://travis-ci.com/conda-forge/miniforge.svg?branch=master)](https://travis-ci.com/conda-forge/miniforge)
 
-This repository holds a minimal installer for conda on platforms that conda-forge supports but that aren't supported by [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
-
-Relevant conversations:
-
-- https://github.com/conda-forge/conda-forge.github.io/issues/871#issue-496677528
-- https://github.com/conda-forge/conda-forge.github.io/pull/922
+This repository holds a minimal installer for conda specific to conda-forge. It can be directly compared to [Miniconda](https://docs.conda.io/en/latest/miniconda.html), with the added feature that conda-forge is the default channel.
```
Comment (rgommers) [inline README.md]: Is this change needed? It removes the "... that aren't supported by Miniconda" part, which is not (yet?) true.

Comment (hmaarrfk) [reply README.md]: It's a little forward looking considering many people want a conda-forge miniconda. 

Even though, I'm against it, I might merge in the PR that would make x86 builds on linux
